sympy/solvers/solveset.py

```diff
@@ -739,7 +741,7 @@ def solveset(f, symbol=None, domain=S.Complexes):
         A set of values for `symbol` for which `f` is True or is equal to
         zero. An `EmptySet` is returned if `f` is False or nonzero.
         A `ConditionSet` is returned as unsolved object if algorithms
-        to evaluatee complete solution are not yet implemented.
+        to evaluate complete solution are not yet implemented.
```

Comment:
> Ok. Shall I revert this?

No. It is generally advised to avoid unrelated changes because review of one change can stall merging of other. Instead, lots of small PRs are recommended, as they are easier to manage, review and merge. This is a really small uncontroversial change, there is no need to "unfix" that change just to create another PR.